src/openms/include/OpenMS/FORMAT/MSstatsFile.h

```diff
@@ -65,7 +65,7 @@ namespace OpenMS
 
         /// store label free experiment (MSstats)
         void storeLFQ(const String& filename, 
-                      const ConsensusMap &consensus_map,
+                      ConsensusMap &consensus_map, // we might add singleton protein groups
```

Comment:
is that a good idea? I would not expect that storing my data will change it. This violates my intuition!

Reply:
hmm true. is there an easy way to do that without copying the full consensusMap?

Reply:
Well we just have to assume that every protein is found in a group, too. I.e. singletons are present. Or calculate and store the singletons outside of the "global" datastructures.